macrosynergy/signal/signal_return_relations.py

```diff
@@ -18,6 +22,8 @@
 )
 import macrosynergy.visuals as msv
 
+np.seterr(divide="ignore")
```

Comment:
I'd suggest we use `with np.errstate(divide='ignore'):` rather than disabling it globally